src/generics/services/user.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * name : user.js
+ * author : Mallanagouda R Biradar
+ * Date : 16-July-2024
+ * Description : All user functions.
+ */
+
+const request = require('request')
+const userServiceUrl = process.env.USER_SERVICE_URL
+
+/**
+ * Retrieves user roles based on the provided filter data.
+ * @param {Object} filterData - The filter data containing the entityTypeId.
+ * @param {string} filterData.entityTypeId - The ID of the entity type to filter user roles.
+ * @returns {Promise<Object>} A promise that resolves to the response containing the fetched user roles or an error object.
+ */
+const userRole = function (filterData) {
+	return new Promise(async (resolve, reject) => {
+		try {
+			// Construct the URL for the user service
+			let url = userServiceUrl + CONSTANTS.common.USER_LIST
```

Comment:
@MallanagoudaB The endpoint variable name is not suiting, you are not calling user_list api